src/daemon/scheduler.rs

```diff
@@ -206,16 +177,10 @@ fn requeue_time_test() -> Time {
     Time::now() + chrono::Duration::seconds(5)
 }
 
-async fn try_publish(
-    event_queue: &Arc<MessageQueue>,
-    ca_manager: Arc<CaManager>,
-    repo_manager: Option<Arc<RepositoryManager>>,
-    ca: Handle,
-) {
-    info!("Try to publish for '{}'", ca);
-    let publisher = CaPublisher::new(ca_manager.clone(), repo_manager);
+async fn try_sync_repo(event_queue: &Arc<MessageQueue>, ca_manager: Arc<CaManager>, ca: Handle) {
```

Comment:
Was publishing not the right thing to communicate to the user here then?

Reply:
imo it was too noisy, we now have:     debug!("Synchronize CA {} with repository", ca);

If there are issues there will be log lines. And a status which can be checked.

Reply:
I can make it `info!` again.. no strong feelings

Reply:
Leave it as you wrote it in the PR, I just wondered why you changed it.